src/tests/JIT/Regression/JitBlue/GitHub_36614/GitHub_36614.csproj

```diff
@@ -8,16 +8,10 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="$(MSBuildProjectName).cs" />
+
+    <CLRTestEnvironmentVariable Include="COMPlus_JitStressModeNames" Value="STRESS_RANDOM_INLINE STRESS_DBL_ALN" />
   </ItemGroup>
   <PropertyGroup>
```

Comment:
Nit - feel free to ignore this or leave for a subsequent mop-up PR especially if you'd need to regenerate all the project changes but it would be nice to get rid of the emptied property groups.

Reply:
This is a really good catch.  Most of the empty property groups are gone.  My script did that, but I shouldn't have left it quiet when it found additional lines since my audit obviously missed that.  In a few cases there were comments that should be moved as well.  I'll do that by hand on top of these changes.

Reply:
I noticed that these are not really frequent and are probably just caused by the comments which are presumably valuable and can be moved to the new style environment variable items exactly as you say. Thanks for doing that!

Reply:
Just for reference - https://github.com/dotnet/runtime/pull/76458/commits/24e297aca1b87354b28c2e90fd68f1d993e1f688 does this.  I removed a few comments that said capitalization matters for Linux because it applies to every one of these.